src/m365/spfx/commands/project/project-azuredevops-pipeline-model.ts

```diff
@@ -0,0 +1,38 @@
+export interface azureDevOpsPipeline {
+  name?: string,
+  trigger: {
+    branches: {
+      include: string[];
+    },
+    workflow_dispatch?: any
+  },
+  pool: {
+    vmImage: string
+  },
+  variables: {
+    name: string,
+    value: string
+  }[],
+  stages: {
+    stage: string,
+    jobs: {
+      job: string,
+      steps: azureDevOpsPipelineStep[]
+    }[]
+  }[]
+}
+
+export interface azureDevOpsPipelineStep {
```

Comment:
Interfaces are types. For consistency, let's write them with uppercase first letter